test/cpp/jit/test_lexer.cpp

```diff
@@ -45,7 +45,7 @@ TEST(LexerTest, SlightlyOffIsNot) {
 TEST(LexerTest, SlightlyOffNotIn) {
   std::vector<std::string> suffixes = {"", " ", "**"};
   for (const auto& suffix : suffixes) {
-    std::vector<std::string> extras = {"i", "i3"};
+    std::vector<std::string> extras = {"i", "i3", "inn"};
```

Comment:
Using `auto` to for `std::initializer_list`? The same to `suffixes`

Reply:
it's a test, no need to micro-optimize it